### ISSUE TITLE
Generate and store dataframes for condor raw hadoop metrics

### ISSUE THREAD

**Issue by dciangot** [dmwm/CRABServer#7313]
- [ ] definition of the contents
- [ ] define the table format to be store
- [ ] evaluate the amount of data we are talking about
- [ ] insert the information in a MONIT backend to be defined

**Comment by dciangot** [comment]
a minimum set of information for visualizing would be something like: cpueff/walltime/failure rate of jobs by input data tier/site/mc production/offsite vs onsite

**Comment by belforte** [comment]
@dciangot would it be worth to consider persising the dataframes (pandas ? we should define a format) in a way where they can be used outside grafana (matplot ?) in case we want more flexibility

**Comment by dciangot** [comment]
Yes, I was thinking to a similar option. I still to figure out where would be easier to use, but let's put it explicitly in the list before forgetting :)

**Comment by nutty7fold** [comment]
check the draft of this issue here -> https://github.com/nutty7fold/cern-crab-data-analysis/tree/main/issue7313

**Comment by belforte** [comment]
@dciangot should we assign this to Rahul ? Does he have a GH account ?

**Comment by dciangot** [comment]
@dynamic-entropy sure
